src/history/hs.c

```diff
@@ -554,6 +554,37 @@ __hs_calculate_full_value(WT_SESSION_IMPL *session, WT_ITEM *full_value, WT_UPDA
     return (0);
 }
 
+/*
+ * __hs_second_update_after_prepare --
+ *     Get the second update that is not a tombstone after the prepared update if it exists
+ */
+static WT_UPDATE *
+__hs_second_update_after_prepare(WT_UPDATE *upd)
```

Comment:
I have a confusion with before after terminology. Could you change it as "__hs_second_older_update_to_prepare"